3-0-spring-mvc/3-0-1-hello-spring-mvc/src/main/java/com/bobocode/mvc/api/NoteRestController.java

```diff
@@ -0,0 +1,23 @@
+package com.bobocode.mvc.api;
+
+/**
+ * To make the controller work, you have to add some annotations:
+ * - @RestController - to let spring know that you want make a REST controller;
+ * - @RequestMapping("/api/notes") - to indicate the basic endpoint for all request methods;
+ * - @RequiredArgsConstructor (for using storage without explicit autowiring);
+ */
+public class NoteRestController {
+
```

Comment:
@erneman1 Spring DI is not in the scope of this exercise, so let's make life easier for people and **inject** `Notes` for them.